### ISSUE TITLE
requirement attributes in html elements

### ISSUE THREAD

**Issue by mxfactorial** [mxfactorial/requirehub-web#13]
Steps:
1. Add a custom `requirement` attribute and [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Special_case_styles) value to every html element fulfilling an explicit requirement. For example, the input element on [line 28](https://github.com/mxfactorial/requirehub-web/blob/28738bf6e4b602abae9a97144c10c668cb581c7c/src/components/Requirement/RequirementForm.js#L28) of the RequirementForm.js component receives a `requirement` attribute assigned to the explicitly-defined `title-input` value in [addRequirement](https://docs.google.com/drawings/d/1x8-lowjEk_KuDna-m8LRiy-vjzuFDb_-2ZXwx5Tjq-c/edit?usp=sharing).

Expected:
1. 
```
<input type="text"
 requirement="title-input"
 name="title"
 onChange={this.handleUpdate}
 placeholder="Title"/>
```
2. Test frameworks reference the designated `requirement` selector during execution to demonstrate for repo owner test automation coverage of requirements (e.g. `let titleInput = await page.$('[requirement=title-input]');` using [puppeteer api](https://github.com/GoogleChrome/puppeteer/blob/ffe5b63dbade012744f6a91a6f9a89b3615d5246/docs/api.md#pageselector)).

**Comment by enesTufekci** [comment]
Is there anything waiting to be done to close this?

**Comment by mxfactorial** [comment]
Yes, the custom requirement attribute does not [currently exist](https://github.com/mxfactorial/requirehub-web/tree/234834193d0c52d6cbbfbb80d4475ecc36d69c16) in any html. Self-assigning to free you up for other tasks: https://github.com/mxfactorial/requirehub-web/tree/ISSUES-13-requirement-attributes

**Comment by mxfactorial** [comment]
Came across a requirement attribute [you added on March 2nd](https://github.com/mxfactorial/requirehub-web/blob/234834193d0c52d6cbbfbb80d4475ecc36d69c16/src/components/Requirement/RequirementForm.js#L35), but doesn't appear in GitHub search. Anyway, adding others in dev branch.
